### PR TITLE
Apple TV

### PR DESCRIPTION
**Pull request by kondranton**
# Apple TV 
Must-have features:
* Possibility to open user's courses
* Possibility to browse courses & search
* Watch video steps
* Read theory steps & quiz text

### CODE REVIEW

File: Stepic/AuthInfo.swift [hunk 1]
```diff
@@ -41,7 +41,7 @@ class AuthInfo: NSObject {
                         }
                         print("downloaded user")
                         CoreDataHelper.instance.save()
-
+
```
Comment (beaverkiria) [inline Stepic/AuthInfo.swift]: plz remove white spaces

File: Stepic/AuthInfo.swift [hunk 2]
```diff
@@ -7,28 +7,28 @@
...
-//                    CoreDataHelper.instance.save()
-//                }
+                //                if c > 1 {
+                //                    print("users count > 1, deleting all")
+                //                    for user in users {
+                //                        user.MR_deleteEntity()
+                //                    }
+                //                    c = 0
+                //                    CoreDataHelper.instance.save()
+                //                }
```
Comment (beaverkiria) [inline Stepic/AuthInfo.swift]: plz remove white spaces

File: Stepic/AuthManager.swift [hunk 3]
```diff
@@ -104,7 +104,7 @@ class AuthManager : NSObject {
                 json = response.result.value!
             }
             let response = response.response
-
+
```
Comment (beaverkiria) [inline Stepic/AuthManager.swift]: plz remove white spaces

File: Stepic/ConnectionHelper.swift [hunk 4]
```diff
@@ -67,7 +67,9 @@ class ConnectionHelper : NSObject {
                 completed, errors in 
                 print("Cancelled \(completed) videos")
                 if completed + errors != 0 {
+                    #if os(iOS)
                     UIThread.performUI({Messages.sharedManager.showCancelledDownloadMessage(count: completed)})
```
Comment (beaverkiria) [inline Stepic/ConnectionHelper.swift]: plz fix formatting

Comment (beaverkiria) [inline Stepic/ConnectionHelper.swift]: plz fix formatting

File: Stepic/DeepLinkRouter.swift [hunk 5]
```diff
@@ -31,33 +32,37 @@ class DeepLinkRouter {
...
             
             if components.count == 3 {
+                
                 AnalyticsReporter.reportEvent(AnalyticsEvents.DeepLink.course, parameters: ["id": courseId as NSObject])
+                
                 routeToCourseWithId(courseId, completion: completion)
                 return
             }
-    
+
```
Comment (beaverkiria) [inline Stepic/DeepLinkRouter.swift]: plz remove white spaces

File: Stepic/DeepLinkRouter.swift [hunk 6]
```diff
@@ -188,22 +193,22 @@ class DeepLinkRouter {
...
             {
-                errorMsg in 
+                errorMsg in
                 print(errorMsg)
                 completion(nil, false)
-            }
+        }
         )
-
+
```
Comment (beaverkiria) [inline Stepic/DeepLinkRouter.swift]: plz remove white spaces

**Comment by Ostrenkiy** [comment]
Вмердживаю первую итерацию Apple TV приложения в development/appleTV

**Pull request merged**